### PR TITLE
Outreachy week nine/update android tests

### PR DESCRIPTION
**Pull request by Lamouresparus**
- Updated tests strings, integers, and doubles to constants

### CODE REVIEW

File: services_app/src/androidTest/java/org/opendatakit/database/service/OdkDatabaseServiceImplTest.java [hunk 1]
```diff
@@ -65,6 +65,16 @@
...
    public static final int COLUMN_ID2_VALUE_ONE = 3;
+   public static final double TEST_DECIMAL_ONE = 3.1415;
+   public static final double TEST_DECIMAL_TWO = 9.5;
+   public static final double TEST_DECIMAL_THREE = 9.813793;
+   public static final int TEST_INT_ONE = 16;
+   public static final int TEST_INT_TWO = 15;
+   public static final int TEST_INT_THREE = 18;
+   public static final int TEST_INT_FOUR = 2;
+   public static final String TEST_STRING_ONE = "test2";
+   public static final String TEST_STRING_TWO = "columnId = ?";
```
Comment (wbrunette) [inline services_app/src/androidTest/java/org/opendatakit/database/service/OdkDatabaseServiceImplTest.java:76]: This is actually the where clause so the name COLUMN_ID_WHERE_CLAUSE is more appropriate, it is not a test string

File: services_app/src/androidTest/java/org/opendatakit/database/service/OdkDatabaseServiceImplTest.java [hunk 2]
```diff
@@ -290,54 +300,54 @@ private void verifyRowExistsInLocalTable(int expectedNumRows, BaseTable baseTabl
...
-             .insertLocalOnlyRow(APPNAME, dbHandle, TABLE_LOCAL, makeTblCvs("a", 2, 3.1415));
+             .insertLocalOnlyRow(APPNAME, dbHandle, TABLE_LOCAL, makeTblCvs(TEST_STRING_THREE, 2, TEST_DECIMAL_ONE));
          baseTable = serviceInterface
              .simpleQueryLocalOnlyTables(APPNAME, dbHandle, TABLE_LOCAL, null, null, null, null,
                  null, null, null, null);
 
-         verifyRowExistsInLocalTable(3, baseTable, "test", 16, 9.5);
-         verifyRowExistsInLocalTable(3, baseTable, "test2", 15, 3.1415);
-         verifyRowExistsInLocalTable(3, baseTable, "a", 2, 3.1415);
+         verifyRowExistsInLocalTable(3, baseTable, TEST_USERNAME, TEST_INT_ONE, TEST_DECIMAL_TWO);
```
Comment (wbrunette) [inline services_app/src/androidTest/java/org/opendatakit/database/service/OdkDatabaseServiceImplTest.java:336]: The TEST_USERNAME is not the same conceptually as TEST_STRING. While the values might be the same it should be kept separate as if we change the username, we may not want to change the test string.

**Review by wbrunette — CHANGES_REQUESTED** [review]

File: services_app/src/androidTest/java/org/opendatakit/database/service/OdkDatabaseServiceImplTest.java (hunk 1 above)
Comment (wbrunette) [reply services_app/src/androidTest/java/org/opendatakit/database/service/OdkDatabaseServiceImplTest.java:76]: here is some info https://www.w3schools.com/sql/sql_where.asp

Comment (Lamouresparus) [reply services_app/src/androidTest/java/org/opendatakit/database/service/OdkDatabaseServiceImplTest.java:76]: Thank you. I understand now. I have renamed it.

File: services_app/src/androidTest/java/org/opendatakit/database/service/OdkDatabaseServiceImplTest.java (hunk 2 above)
Comment (Lamouresparus) [reply services_app/src/androidTest/java/org/opendatakit/database/service/OdkDatabaseServiceImplTest.java]: Okay. I have made the change. Thanks

**Comment by wbrunette** [comment]
runtests

**Review by wbrunette — APPROVED** [review]

**Pull request merged**